src/components/entries/EntriesContainer.js

```diff
@@ -88,8 +90,8 @@ export default createClass({
       return <p>No entries are available.</p>
     } else {
```

Comment:
You're `return`ing in both `if` and `else if` clauses. You can get rid of `else` as the code here is rather long.